client/src/hooks/networks/index.ts

```diff
@@ -13,6 +17,144 @@ export type NetworkResponse = {
   isError: boolean;
 };
 
+export type OrganizationProperties = {
+  id: number | undefined;
+  name: string | undefined;
+  type: 'organization';
+  countryLat: number;
+  countryLong: number;
+};
+
+export type ProjectProperties = {
+  id: number | undefined;
+  name: string | undefined;
+  type: 'project';
+  countryLat: number;
+  countryLong: number;
+};
+
+type GroupedNetworks = {
+  [key: string]: (OrganizationProperties | ProjectProperties)[];
+};
+
+type NetworkProperties = {
+  countryName: string;
+  organizations: OrganizationProperties[];
+  projects: ProjectProperties[];
+};
+
+type PointFeatureWithNetworkProperties = PointFeature<NetworkProperties>;
+
+export type NetworkMapResponse = {
+  features: PointFeatureWithNetworkProperties[];
+  isFetching: boolean;
+  isFetched: boolean;
+  isPlaceholderData: boolean;
+  isError: boolean;
+};
+
+export const useMapNetworks: () => NetworkMapResponse = () => {
+  const {
+    data: organizationsData,
+    isFetching: organizationIsFetching,
+    isFetched: organizationIsFetched,
+    isPlaceholderData: organizationIsPlaceholderData,
+    isError: organizationIsError,
+  } = useGetOrganizations({
+    populate: 'country',
+  });
+
+  const {
+    data: projectsData,
+    isFetching: projectsIsFetching,
+    isFetched: projectsIsFetched,
+    isPlaceholderData: projectsIsPlaceholderData,
+    isError: projectsIsError,
+  } = useGetProjects({
+    populate: 'country_of_coordination',
+  });
```

Comment:
You need to load *all* the organisations and projects on the map. It seems that some are missing.

For example, “Agricultural Producers Group TERRA” from Madagascar is not showing.